addons/mail/static/src/views/fields/statusbar_duration/statusbar_duration_field.js

```diff
@@ -13,7 +13,10 @@ export class StatusBarDurationField extends StatusBarField {
         const durationTracking = this.props.record.data.duration_tracking || {};
         if (Object.keys(durationTracking).length) {
             for (const item of items) {
-                const duration = durationTracking[item.value];
+                let duration = durationTracking[item.value];
+                if (duration == 0) {
+                    duration = 1;
+                }
```

Comment:
```
let duration = durationTracking[item.value];
duration = duration ? duration : 1;
```
but then, do we not alway enter the if below ? Could we have negative values ?

Reply:
Yes, we will always enter the if over here. I have added this code to get `1m` as soon as i click on next stage. If i don't add it we will not get any time on next stage as soon as i click on it.
We can also remove it if we don't want `1m` as soon as i click on next stage.

I will change my code and add the ternary operator as you have suggested.
Thanks!!

Reply:
For above we are having 3 cases - 

1. For the upcoming stages : we will get duration null so it will go to else and `item.shortTimeInStage will be set to 0` so that we does not get any tracking of it.

2. For previous stages or current stage : we will get duration a positive value so it will go in if(duration>0) and `item.shortTimeInStage = formatDuration(duration, false);` so that we will get tracking of it.

3. For next stage on which we are clicking : we will get duration=0 so it will also go in else and `item.shortTimeInStage will be set to 0` so that we does not get any tracking of it.

But after the fix - 
```
let duration = durationTracking[item.value];
duration = (duration==0) ? 1 : duration;
```

First two case will remain same but for 3 case - we will get duration=0 which will be converted to 1 and then it will go in if(duration>0) and `item.shortTimeInStage = formatDuration(duration, false);` so that we will get tracking of it as `1m`.

I hope it might help you to understand. :slightly_smiling_face: